pkg/kp/consulutil/consultest.go

```diff
@@ -1,7 +1,7 @@
-// The consultest package contains common routines for setting up a live Consul server for
+// This file contains common routines for setting up a live Consul server for
```

Comment:
If you're changing this, I'd prefer it be changed to match the convention. The godoc spec decrees this style: "package consulutil contains"